### DISCUSSION TITLE
Error in plugin @nomiclabs/hardhat-etherscan: The contract verification failed. Reason: Fail - Unable to verify

### DISCUSSION THREAD

**Discussion by progressaded** [hashlips-lab/nft-erc721-collection#374] [General]
Hello everyone  ,

I am having trouble verifying my smart contract , I have installed everything as shown in the "ERC721 Collection 101 - Basic setup on Windows 10" video from Liarco and I get this error when trying to verify my contract
![error](https://user-images.githubusercontent.com/107586541/182718308-96cce23c-dee9-473d-9d93-084e9dd538a3.png)
 Could someone help me please ?

Thanks in advance and have a nice day

**Comment by locache** [comment]
are you trying to verify it via command line? If so what is the command you are using? Also, are you deploying to a blockchain testnet? If so, which one?
